### ISSUE TITLE
Recipe / API: pluck / map after a join

### ISSUE THREAD

**Issue by mglukhovsky** [rethinkdb/docs#25]
- API docs for `eq_join`, `inner_join`, and `outer_join` doesn't describe how to use `zip` to combine the output
- In the [joins documentation](http://rethinkdb.com/docs/table-joins/), we should explain how to use `pluck` and `map`

**Comment by chipotle** [comment]
The `eq_join` documentation now goes into some detail about `zip`. The examples for `outer_join` and `inner_join` *use* `zip` but don't explain it, they just link to it. However, I'm not sure it's worth elaborating much on it, given that we advise not using these commands and there's still a "modernizing joins" proposal open that might get rid of those commands entirely.

Given that there's a whole separate document on Map-Reduce, I don't think we should explain how to use `map` in the Table Joins document, and I'm not sure why we'd need to elaborate on `pluck` there specifically.

tl;dr: closing. Reopen something more specific if it seems like a good idea.
